_posts/2023-04-19-Announcing-Data-Prepper-2.2.0.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: post
+title:  "Announcing Data Prepper 2.2.0"
+authors:
+- dlv
+date: 2023-04-19 14:30:00 -0500
+categories:
+  - releases
+meta_keywords: Data Prepper 2.2.0
+meta_description: Data Prepper 2.2.0 improves data delivery assurances with end-to-end acknowledgements and an S3 DLQ.
+---
+
+Data Prepper 2.2.0 is now available for [download](https://opensearch.org/downloads.html#data-prepper)!
+This release has a number of changes that help with Data Prepper’s reliability and data delivery assurances.
+
+## S3-based dead-letter queue for OpenSearch
+
+Prior to Data Prepper 2.2.0, the `opensearch` sink could only write failed events to a local file. 
+This required logging into your cloud instance or machine to retrieve failed events as well creating a different infrastructure to export them.
+
+Now, the `opensearch` sink can write documents from failed events directly in to an Amazon S3 dead-letter-queue (DLQ). 
+This helps you analyze event failures without having to retrieve them locally. 
+Furthermore, users that run on a serverless infrastructure can avoid maintaining a persistent fail state on serverless machines.
+
+## End-to-end acknowledgements for S3 source
+
+Data Prepper's `s3` source now support end-to-end acknowledgments.
+
+Before end-to-end acknowledgments, the `s3` source would only acknowledge event delivery with Amazon Simple Queue Service (SQS) after writing all events to a Data Prepper buffer. 
+In cases where Data Prepper was unable to write to OpenSearch, the SQS message would still be acknowledged, and Data Prepper would not read for the object.
+
+With end-to-end acknowledgments, the `s3` source does not acknowledge completion until all events are sent to an OpenSearch index or the `opensearch` sink's DLQ. 
+If the `s3` source receives no acknowledgment, the SQS message remains in the SQS queue for processing again.
+
+For Data Prepper 2.2.0, end-to-end acknowledgments are only supported inside the `s3` source since acknowledgments to `s3` are asynchronous. 
```

Comment:
```suggestion
For Data Prepper 2.2.0, end-to-end acknowledgments are only supported inside the `s3` source because acknowledgments to `s3` are asynchronous. 
```